packages/maskbook/src/social-network-adaptor/minds.com/collecting/identity.ts

```diff
@@ -0,0 +1,29 @@
+import { ProfileIdentifier } from '../../../database/type'
+import { creator, SocialNetworkUI as Next } from '../../../social-network'
+import { mindsBase } from '../base'
+import { selfInfoSelectors } from '../utils/selector'
+
+function resolveLastRecognizedIdentityInner(
+    ref: Next.CollectingCapabilities.IdentityResolveProvider['lastRecognized'],
+    cancel: AbortSignal,
+) {
+    const handle = selfInfoSelectors().handle.evaluate()
+
+    fetch('/api/v1/channel/' + handle, { signal: cancel })
```

Comment:
a one-time api access might be OK but is it possible to get those information without a request?

Reply:
It's possible to get username and avatar url from the page but not the name. That's why I resorted to this.